template/page-ambassadors.html.jinja

```diff
@@ -413,6 +268,161 @@
+                    <img src="./assets/img/ambassadors/quote2-small.svg" class="h-150 d-lg-none  mb-4">
+                    <img src="./assets/img/ambassadors/quote2.svg" class="h-100 d-none d-lg-block">
+                    <div class="p-0 col-lg-7 mx-lg-auto">
+                        <p class="p-lg-3 mb-2"><strong>Sally Z.</strong><br >
+                            Toronto Metropolitan University<br >
+                            Spring 2023 XRPL Campus Ambassador</p>
+                    </div>
+                </div>
+            </div>
+          </div>
+        
+         <div class="carousel-item mb-40">
+            <div class="p-0">
+                <div class="mb-4 p-lg-3">
+                    <img src="./assets/img/ambassadors/quote3-small.svg" class="h-150 d-lg-none  mb-4">
+                    <img src="./assets/img/ambassadors/quote3.svg" class="h-100 d-none d-lg-block">
+                    <div class="p-0 col-lg-7 mx-lg-auto">
+                        <p class="p-lg-3 mb-2"><strong>Nick D.</strong><br >
+                            Miami University<br >
+                            Spring 2023 XRPL Campus Ambassador</p>
+                    </div>
+                </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    
+
+</section>
+
+<!-- How it Works -->
+<section class="container-new py-26">
+    <!-- flex. Col for mobile. Row for large. on large align content to the center  -->
+    <div class="d-flex flex-column flex-lg-row align-items-lg-center">
+
+        <div class="order-1 mr-lg-4 mb-4 pb-3 mb-lg-0 pb-lg-0 col-lg-6 px-0">
+            <div class="d-flex flex-column-reverse p-lg-3">
+                <h3 class="h4 h2-sm">{% trans %}Process to become a Campus Ambassador{% endtrans %}</h3>
+                <h6 class="eyebrow mb-3">{% trans %}How it Works{% endtrans %}</h6>
+            </div>
+
+            <p class="p-lg-3 mb-2 longform">{% trans %}Apply now to become an XRPL Campus Ambassador.{% endtrans %}</p>
+
+            <div class="d-none d-lg-block p-lg-3">
+                <button data-tf-popup="rTioxhMY" data-tf-iframe-props="title=XRPL Campus Ambassador" data-tf-medium="snippet" 
+                    class="btn btn-primary btn-arrow-out"
+                    data-tf-hidden="utm_source=xxxxx,utm_medium=xxxxx,utm_campaign=xxxxx,utm_term=xxxxx,utm_content=xxxxx">Apply for Fall 2023</button>
+            </div>
+        </div>
+
+        
+        <div class="order-2 col-lg-6 px-0 ml-lg-2">
+
+            <div class="row m-0">
+
+                <div class="col-12 col-lg-6 p-0 pr-lg-4">
+                    
+                    <div class="px-lg-3 pb-3">
+                        <img src="./assets/img/ambassadors/01.svg" class="pl-lg-3">
+                        <div class="p-lg-3 pt-3">
+                            <h6 class="mb-3">Apply</h6>
+                            <p class="">Submit an application to be considered for the Campus Ambassador program.</p>
```

Comment:
For this and all the following instances:

- Omit the `class` attribute if it's empty
- Add `{% trans %}` tags for user-facing text